playbook.md

```diff
@@ -2230,6 +2230,8 @@ Slack channel.
 
 ### Benefits
 
+Subject to successful probation.
```

Comment:
Would it be clearer to make it into a full sentence, such as "Benefits start applying on the day" (or "after the day", I don't know which) "you pass your probation" ?

Reply:
Yes, Something like 'All perks and benefits are available to the dxw team after the successful passing of their probationary period.'